modules/bricksBidAdapter.js

```diff
@@ -0,0 +1,312 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please just import the needed function here. We no longer have the root 'utils.' in other adapters.